### ISSUE TITLE
new timoni module has files listed in the timoni.ignore

### ISSUE THREAD

**Issue by verdverm** [stefanprodan/timoni#158]
looks like 3 files:
- timoni.ignore
- go.{mod,sum}

![image](https://github.com/stefanprodan/timoni/assets/1390600/eba072e9-de48-4a0e-9bbb-7f5abe852a51)

**Comment by stefanprodan** [comment]
Yes these are the new default ignore rules to avoid publishing modules to OCI with files which are not required at runtime.

**Comment by verdverm** [comment]
The problem is that the generate go files (and cue mod setup) would have over written my existing files

if I generated a new timoni module at the root of my existing repository

**Comment by stefanprodan** [comment]
The `timoni mod init mymod .` creates a directory with the module name, if the dir exists it should error out. How did you run the command so it ended up overriding files?

**Comment by stefanprodan** [comment]
The files needed for a Timoni module are listed here https://timoni.sh/module/

**Comment by verdverm** [comment]
What's up with the generated go.mod/sum files then? They are not in the list, I'm not even in a project containing any Go

**Comment by stefanprodan** [comment]
When you do `timoni mod push` those Go files are ignored as per `timoni.ignore`, they are useful to module authors. As an author you should have them in your Git repo to be able to update the Kubernetes definitions https://timoni.sh/module/#kubernetes-types. If you don’t use `go get` and `cue get go` then you don’t need them.
